drivers/hmis/lib/errors.rb

```diff
@@ -0,0 +1,41 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Errors
+  class CustomValidationError
```

Comment:
I thought one of the rules for the new classloader was 1 class per file?

Reply:
It seems to be working fine, but I can split it to two files